nilearn/plotting/displays.py

```diff
@@ -257,9 +257,13 @@ def transform_to_2d(self, data, affine):
                              self.direction)
         return cut
 
-    def draw_position(self, size, bg_color, **kwargs):
+    def draw_position(self, size, bg_color, decimal=False, **kwargs):
```

Comment:
Since this function is in the public API, we should take this opportunity to write docstrings...